sample/AirshipSample/ios/AirshipSample Carthage-Info.plist

```diff
@@ -36,9 +38,7 @@
 		</dict>
 	</dict>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>Urban Airship React Native sample location, when in use</string>
-	<key>NSLocationAlwaysUsageDescription</key>
-	<string>Urban Airship React Native sample location</string>
+	<string></string>
```

Comment:
Seems odd to have the key with an empty string value. Why not just remove the key and value?

Reply:
broke the single info plist in two to account for the two targets, re-added these keys